cmd/kube-proxy/app/server_others.go

```diff
@@ -326,7 +326,7 @@ func getProxyMode(proxyMode string, khandle ipvs.KernelHandler, ipsetver ipvs.IP
 	case proxyModeIPVS:
 		return tryIPVSProxy(khandle, ipsetver, kcompat)
 	}
-	klog.Warningf("Flag proxy-mode=%q unknown, assuming iptables proxy", proxyMode)
+	klog.Warningf("Value of proxy mode: %q unknown, it may be overwritten by config file if flag --config is added, assuming iptables proxy", proxyMode)
```

Comment:
does the config value really take precedence over the flag value?

Reply:
i would change this to:
`Unknown proxy mode %q, assuming iptables proxy`

users would still know where to set the mode.

Reply:
The config value will take precedence over the flag.

Reply:
yeah, this is counter intuitive.
i would still recommend changing this line to `Unknown proxy mode %q, assuming iptables proxy`

Reply:
Ok, I will recheck it. Thanks for review it.